code/__HELPERS/chat.dm

```diff
@@ -97,3 +97,6 @@ it will be sent to all connected chats.
 	for(var/mob/creature as anything in mob_list)
 		to_chat(creature, message)
 	send_to_observers(message, source)
+
+/// Emboldens runechat messages
+#define RUNECHAT_BOLD(str) "+[str]+"
```

Comment:
global defines should go in `__DEFINES`, not `__HELPERS`. there should be a big list of `span_x()` macros somewhere in there, that file might work? there's probably another chat.dm there though

Reply:
Ah alright, thanks
I've moved it to __DEFINES\chat.dm if that's okay